macros/edr/materializations/model/incremental.sql

```diff
@@ -62,7 +62,7 @@
   {% do return(relations) %}
 {% endmaterialization %}
 
-{% materialization incremental, adapter="athena", supported_languages=["sql"] %}
+{% materialization incremental, adapter="athena", supported_languages=["sql", "python"] %}
```

Comment:
Athena added support for python models and because of us they don't work, so added here to the supported languages.
Added to trino as well because there's actually no harm in doing so (we did the same for redshift even though it doesn't yet support Python models)